webpack/assets/javascripts/react_app/components/CommonParameterForm/components/__tests__/HiddenValueField.test.js

```diff
@@ -0,0 +1,9 @@
+import { testComponentSnapshotsWithFixtures } from '../../../../common/testHelpers';
```

Comment:
I think it's better to import it from `@theforeman/test`

```suggestion
import { testComponentSnapshotsWithFixtures } from '@theforeman/test';
```

where it's could get updated and more maintained,
can you update the rest of these imports? thanks